src/Directory.Build.props

```diff
@@ -1,27 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-
-  <PropertyGroup>
-    <VersionPrefix>3.6.2</VersionPrefix>
-    <TargetFramework>net7.0</TargetFramework>
-    <RootNamespace>Neo.Plugins</RootNamespace>
-    <Authors>The Neo Project</Authors>
-    <PackageTags>NEO;Blockchain</PackageTags>
-    <PackageProjectUrl>https://github.com/neo-project/neo-modules</PackageProjectUrl>
-    <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <RepositoryType>git</RepositoryType>
-    <RepositoryUrl>https://github.com/neo-project/neo-modules.git</RepositoryUrl>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <None Include="config.json" Condition="Exists('config.json') And '$(IncludeSettingsFileOutput)' != 'False'">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
-    </None>
-  </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Neo" Version="3.6.2" />
-  </ItemGroup>
-
-</Project>
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
+	<ItemGroup>
+		<None Include="config.json" Condition="Exists('config.json') And '$(IncludeSettingsFileOutput)' != 'False'">
```

Comment:
We've excluded `IncludeSettingsFileOutput` from every project, so I think we don't need this condition (`'$(IncludeSettingsFileOutput)' != 'False'`) to be presented here, it's just not relevant anymore.

Is `config.json` a configuration file of the module's dependent library? If so, then we probably just don't need to copy it while publishing module.

Reply:
> We've excluded `IncludeSettingsFileOutput` from every project

Not every project, just part of them.

Reply:
I mean from every neo-modules project. We don't have any mentions of `IncludeSettingsFileOutput=False` anymore in neo-modules.